etc/profile-m-z/ssh.profile

```diff
@@ -19,7 +19,7 @@ include disable-exec.inc
 include disable-programs.inc
 
 whitelist ${RUNUSER}/gcr/ssh
-whitelist ${RUNUSER}/gnupg/S.gpg-agent.ssh
```

Comment:
Removing this whitelist will cause breakage on systems where that path _is_ indeed the one that is being used (as shown by gpgconf). For now it would be better to just `add` a whitelist for other paths instead of `replacing` known good whitelists.

Reply:
Understood, I allowed for both conditions in the latest commit. And wrote an explanation as a comment.